dev/com.ibm.ws.jdbc_fat/test-applications/basicfat/src/basicfat/DataSourceTestServlet.java

```diff
@@ -1214,6 +1214,24 @@ public void testEnableSharingForDirectLookupsFalse() throws Exception {
         }
     }
 
+    /**
+     * Verify that connections are CONTAINER auth when enableContainerAuthForDirectLookups=true
+     */
+    public void testEnableContainerAuthForDirectLookupsTrue() throws Exception {
+        DataSource ds = (DataSource) new InitialContext().lookup("jdbc/dsfat12");
+        Connection con = null;
+        try {
+            con = ds.getConnection();
+            // user should be dbuser2
+            DatabaseMetaData metadata = con.getMetaData();
+            String user = metadata.getUserName();
+            if (!"dbuser2".equalsIgnoreCase(user))
+                throw new Exception("Connection is using APPLICATION res-auth - enableContainerAuthForDirectLookups property is not being honored. ");
+        } finally {
+            con.close();
```

Comment:
will cause NullPointerException if the getConnection fails, obscuring the real failure

Reply:
done.